Dockerfile

```diff
@@ -0,0 +1,46 @@
+FROM ubuntu:16.04 
+USER root
+
+WORKDIR /tests
+
+ARG TEST_CONFIG="{'error':'missing_config'}"
+
+ENV MOZ_HEADLESS="1"
+
+RUN apt-get -y update && \
+    apt-get install -y -qq software-properties-common &&\
+    apt-get install -y -qq wget unzip \ 
+                       libreadline-gplv2-dev libncursesw5-dev \
+                       libssl-dev libsqlite3-dev tk-dev libgdbm-dev \
+                       libc6-dev libbz2-dev libgtk-3-0 libdbus-glib-1-2 \
+                       libgtk-3-dev libglib2.0-dev &&\
+                       rm -rf /var/lib/apt/lists/* 
```

Comment:
This looks _sneaky_. Since it's a new command (note `&&`), I think we should de-indent it to align with the `apt-get install` commands above (versus aligned to the library names).